src/test/scala/ai/privado/policyEngine/PolicyTests.scala

```diff
@@ -73,31 +75,92 @@ class PolicyTests extends AnyWordSpec with Matchers with BeforeAndAfterAll {
       "",
       Language.JAVASCRIPT,
       Array()
+    ),
+    RuleInfo(
+      "ThirdParties.SDK.Sentry",
+      "Sentry",
+      "",
+      Array("sentry.io"),
+      List("(?i).*(sentry).*"),
+      true,
+      "",
+      Map(),
+      NodeType.REGULAR,
+      "",
+      CatLevelOne.SINKS,
+      "",
+      Language.JAVASCRIPT,
+      Array()
     )
   )
 
-  val policy = PolicyOrThreat(
-    "Policy.Deny.Sharing.LeakToConsole",
-    "Policy to restrict Contact Information being leaked to console",
-    "Example: Don't leak contact data",
-    "Talk to the Data Protection team: dataprotection@org.com",
-    PolicyThreatType.COMPLIANCE,
-    PolicyAction.DENY,
-    DataFlow(List("Data.Sensitive.ContactData.*"), SourceFilter(Option(true), ""), List("Leakages.Log.*")),
-    List(".*"),
-    Map[String, String](),
-    Map[String, String](),
-    "",
-    Array[String]()
-  )
+  "Policy Executor: Leakages" should {
+    val policyLeakages = PolicyOrThreat(
```

Comment:
This Test case looks exactly similar to the one we have below, What are we testing differently Here?